static/docs/commands-reference/gc.md

```diff
@@ -6,7 +6,7 @@ Remove unused objects from cache or remote storage.
 
 ```usage
 usage: dvc gc [-h] [-q | -v] [-a] [-T] [-c]
-              [-r REMOTE] [-f] [-j JOBS]
+              [-r REMOTE] [-f] [-j JOBS] [- R]
```

Comment:
There's a space between `-` and `R`.
But anyway, the usage shown with `dvc gc -h` (latest DVC version: 0.41.3+1e7e2a) is
```
usage: dvc gc [-h] [-q | -v] [-a] [-T] [-c] [-r REMOTE] [-f] [-j JOBS]
              [-p [REPOS [REPOS ...]]]
```
Which doesn't include `-R` (and doesn't match the current docs 😓

Reply:
Ok and it dvc -R is not mentioned in docs for a no of other commands as well like `fetch, status, metrics,gc and somemore`.

Reply:
Thanks. So did you update this usage and remove the `-R` option from this doc?

Could you please open an issue for the docs that are missing the -R option and need update in their usage blocks?

Reply:
Opened issue https://github.com/iterative/dvc/issues/2154_

Reply:
there still a space in this file between - and R

Reply:
@kurianbenoy Where did you get `gc -R` from anyways?